packages/next/app.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "next",
```

Comment:
Does it need to be `next` or can we call it `My App`?  (just wondering)

Reply:
Nope, you can name it whatever you want -- It's just an identifier for the AppRegistry. Actually, I think it may be better to use the package name defined in `package.json` (like Bruno does [here](https://github.com/brunolemos/react-native-web-monorepo/blob/ae4ce9e59ca6fc7adc8f83a3871a78a6327df130/packages/web-nextjs/pages/_document.tsx#L12)) in all of the monorepo packages instead of getting it  from `app.json`. This change should be fine for the web based projects but not sure if changing the identifier is more complicated in the others.